cli/src/command.rs

```diff
@@ -349,7 +350,16 @@ where
 			hwbench,
 		)
 		.map(|full| full.task_manager)
```

Comment:
```suggestion
		.map(|full| full.task_manager)?;
```

`?` is calling `From::from` automatically. This being the reason you need to write `Into::<Error>`